lib/aws-cleaner.rb

```diff
@@ -102,9 +133,9 @@ def parse(body)
   end
 
   # return the instance_id of the terminated instance
-  def process_message(message_body)
-    return false if message_body['detail']['instance-id'].nil? &&
-                    message_body['detail']['state'] != 'terminated'
+  def process_message(message_body, config)
+    return false if message_body['detail']['instance-id'].nil? ||
```

Comment:
This one is my only problem, since we use one queue to monitor the "running" state notifications with aws-watcher and prefer not to add multiple queues since it's more to manage. 

Would it be ok to add a config parameter to delete irelevant messages that is set to true by default, that way  by default the existing behavior is kept but it can be overwritten. I was thinking I can add that either in the main loop or the process_message function.

Reply:
Awesome to see others using that project and the use case makes sense to me.

I'd still prefer it to be in 2 separate PRs though personally @eheydrick what do you think?

Reply:
I have separated the changes. I will submit another pull request for the message handling.